WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -276,6 +276,64 @@ def edit
     render :edit
   end
 
+  def payment_setup
+    @competition = Competition.find(params[:id])
+
+    client = create_stripe_oauth_client
+    oauth_params = {
+      scope: 'read_write',
+      redirect_uri: ENVied.ROOT_URL + '/stripe-connect',
```

Comment:
Could we instead call a Rails URL helper here? Something like `competitions_stripe_connect_url`? I think all this would require is adding a `as: :competitions_stripe_connect` to the corresponding route in our routes file.